inst/schema/SurveyResponseData.schema.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Added this just as an example response type with filters in, and so I can get some test data being passed back to the front end initially